src/portable/ReactUi/data/localization/TranscriberUi-ar.xlf

```diff
@@ -560,5 +560,35 @@
 				<target>معاينة</target>
 			</segment>
 		</unit>
+		<unit id="projectSettings.milestones">
+			<segment>
+				<source>Milestones</source>
+				<target>معالم</target>
+			</segment>
+		</unit>
+		<unit id="projectSettings.start">
+			<segment>
+				<source>Start</source>
+				<target>بداية</target>
+			</segment>
+		</unit>
+		<unit id="projectSettings.transcribed">
```

Comment:
I wonder if we should avoid adding the same string twice in two different places in the localization file? Should we refactor the transcribe.transcribed to use this one or refactor the code around this one to use the other one?